core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -55,6 +56,9 @@ export class PracticeTabComponent implements OnInit {
   }
 
   isStartButtonDisabled(): boolean {
+    if (this.openInTopicEditor) {
```

Comment:
Ideally, avoid the use of verbs in the beginning of variable names. Also, I'm a bit confused about what this means. Can you please explain?